helm/zeebe-test/values.yaml

```diff
@@ -1,35 +1,27 @@
 ph-ee-engine:
-  zeebe:
-    broker:
-      contactpoint: "zeebetest-zeebe-gateway:26500"
 
-  #Old zeebe configuration to be replaced with camunda-platform:zeebe/zeebe gateway:<paramname>
-#  zeebe-cluster-helm:
-#    global:
-#      elasticsearch:
-#        host: "ph-ee-elasticsearch"
-#    image:
-#      repository: camunda/zeebe
-#      tag: 1.1.0
-#
-#    clusterSize: "1"
-#    partitionCount: "1"
-#    replicationFactor: "1"
-#    JavaOpts: "-Xms8g -Xmx8g -XX:+UseParallelGC -XX:MinHeapFreeRatio=5 -XX:MaxHeapFreeRatio=10 -XX:MaxRAMPercentage=25.0 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -XX:+PrintFlagsFinal"
-#
-#    elasticsearch:
-#      enabled: false
-#    kibana:
-#      enabled: false
-#
-#    extraInitContainers: |
-#      - name: init-ph-ee-kafka-exporter
-#        image: busybox:1.28
-#        command: ['/bin/sh', '-c']
-#        args: ['wget -O /exporters/ph-ee-kafka-exporter.jar "https://paymenthub-ee-dev.s3.us-east-2.amazonaws.com/jars/exporter-1.0.0-SNAPSHOT.jar"; ls -al /exporters/']
-#        volumeMounts:
-#        - name: exporters
-#          mountPath: /exporters/
+  camunda-platform:
+    global:
+      zeebe: "zeebetest-zeebe"
+      elasticsearch:
+        disableExporter: true
+    zeebe:
+      broker:
+        contactpoint: "zeebetest-zeebe-gateway:26500"
+      global:
+        elasticsearch:
+          host: "ph-ee-elasticsearch"
+      image:
+        repository: camunda/zeebe
+        tag: 8.1.1
+      extraInitContainers: 
+        - name: init-ph-ee-kafka-exporter
+          image: busybox:1.28
+          command: ['/bin/sh', '-c']
+          args: ['wget -O /exporters/ph-ee-kafka-exporter.jar "https://fynarfin.io/images/exporter-1.1.1-SNAPSHOT.jar"; ls -al /exporters/']
+          volumeMounts:
+          - name: exporters
+            mountPath: /exporters/
```

Comment:
These lines are the primary critical changes but if defaults are set properly in ph-ee-engine, this should be mostly duplicate config.